packages/qb/pqb/src/columns/columnType.test.ts

```diff
@@ -206,6 +206,33 @@ describe('column type', () => {
     });
   });
 
+  describe('.asType', () => {
+    type Type = 'foo' | 'bar';
+    it('should use custom type', () => {
+      const withType = column.asType(columnType<Type>());
```

Comment:
It's hard to choose the look and feel that will stay forever, alternatives to consider:

If it's already a standalone utility that user has to import, it makes sense and is a bit shorter to do it so:
```ts
const withType = setColumnType<Type>(column);
```

But it would look inconsistent as we mix method chaining with wrapping into functions.

---
This looks weird, but with the advantage of not having to import the helper:
```ts
const withType = column.asType((type) => type<Type>());
```
---

How about moving `asType` to the `t` variable? (exported from columnTypes.ts)

Doesn't it look slightly better?

If not really, I'm fine with your current approach.

```ts
columns = this.setColumns((t) => ({
  col: t.asType<Type>(t.text()),
}))
```

Reply:
`column.asType((type) => type<Type>())` (or `column.asType((t) => t<Type>())`) seemed to be the least evil to me.